cluster/gce/gci/configure.sh

```diff
@@ -717,8 +717,7 @@ function install-kube-binary-config {
   # are presented to kubelet:
   # --image-credential-provider-config=${path-to-config}
   # --image-credential-provider-bin-dir=${path-to-auth-provider-binary}
-  # Also, it is required that DisableKubeletCloudCredentialProviders and KubeletCredentialProviders
```

Comment:
Why removing mention of the `DisableKubeletCloudCredentialProviders` ?

Reply:
My mistake.
I added it back.